<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node name="image_publisher" pkg="image_logger" type="image_publisher.py" output="log" required="true">
        <param name="image_topic" value="/image"/>
        <param name="publish_rate" value="0.25"/>
        <param name="image_path" value="$(find image_logger)/images/image-2.jpeg"/>
    </node>
</launch>
